### ISSUE TITLE
Jump to block that tag is attached to

### ISSUE THREAD

**Issue by rdevriese** [gidongkwon/logseq-plugin-tags#20]
Tags plugin lists the specific pages and blocks a tag is attached to. However, if a block is clicked the whole page is displayed but the specific block is not highlighted. If a page contains the same tag on multiple blocks it is difficult to find the specific block that was clicked on in the tags list.

**Comment by tiankonganddahai** [comment]
Yes, recommend add this operation function at next version. When click a tag name in tags list, clearly got the block has this tag.
